### ISSUE TITLE
COATi vs MAFFT for MSA?

### ISSUE THREAD

**Issue by 000generic** [CartwrightLab/coati#28]
Hi!

I'm working on gene and species trees across animals - and have been using MAFFT to produce MSAs having 50-100 species. The paper on COATi was impressive in its performance relative to other tools, including MAFFT - but focused on pairwise alignments. The paper and github highlight future development of COATi for MSAs - and it is an option in the tool on github.

I was wondering if development of COATi for MSAs is stable / mature enough for users to adopted it in their pipelines - or if it is still under active development and not yet recommended for general use.

If it is recommended for general use, do you have a sense of how it compares to other tools, like MAFFT?

Any guidance on this would be greatly appreciated!  

Thank you very much, 
Eric

**Comment by reedacartwright** [comment]
Synopsis: Try it out on a few datasets and let us know if you like the results.

COATi can produce MSAs but could produce better MSAs. We haven't been able to work on that algorithm for about two years, but with another round of grant funding, we would be able to improve things. We haven't tested it as deeply as we have the pairwise algorithm and therefore, I recommend that you try it out to see if you like the alignments that it produces.

**Comment by 000generic** [comment]
Thank you for you quick helpful reply - and sounds good! 

I'm currently running my pipeline with amino acids - but plan to do a DNA version in the coming week or so and can test COATi out at that point for MSAs.  Up until now DNA work wasn't I had considered for deep phylogeny of animal species and gene trees - but inspired by [Kapli 2023](https://academic.oup.com/sysbio/article/72/5/1119/7208165)

Is COATi something that might be expanded to amino acids in the future with funding?

Thank you again!

**Comment by reedacartwright** [comment]
Yes, if some of the novel search algorithms that I am working on work out, we would expand it to amino acids.
